app/views/posts/index.html.erb

```diff
@@ -1 +1,38 @@
-<h1>Here is a list of posts for a given user</h1>
+<h1 class="post-title"><%= @user.name%>'s Posts</h1>
+<section class='user-card-container'>
+  <section class = "user-card">
+    <div class="user-img">
+      <img src="<%= @user.photo %>" width="100px"  height = "100px" class="user-photo">
+    </div>
+    <div class="user-info">
+      <h1><%= @user.name %></h1>
+      <h2 class="post-counter"> Number of posts: <%= @user.posts_counter%></h2>
+    </div>
+  </section>
+  <section class = "user-bio">
+    <p><%= @user.bio %></p>
+  </section>
+  
+  <section class='post-container'>
+    <% @posts.each do |post| %>
+      <%= link_to user_post_path(user_id: post.author_id, id: post.id) do%>
+        <div class="post-card-container">
+          <div class="post-card">
+            <h2><%= post.title %></h2>
+            <p><%= post.text %></p>
+            <div class="reactions">
+              <span>Comments: <%= post.comments_counter %></span>
+              <span>Likes: <%= post.likes_counter %></span>
+            </div>
+          </div>
+          <h4 class="comment-title">Comments</h4>
+          <div class="comment-container">
+            <% post.recent_comments.each do |comment|%>
+              <span><%= comment.user.name %>: <%= comment.text%></span>
+            <% end %>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
+  </section>
+</section>
```

Comment:
- [ ] [REQUIRED] Kindly ensure that you use the `recent_comment` method to render the 5 most recent comments as stated by the project requirement. Also, do not forget to add the pagination button bellow the page. Let's implement these fixes both 👍🏽 

Here is the method in your Post model:
https://github.com/Wineshuga/pen-it/blob/a79f42b6cc38923e429a00fa36bc3dc076adf5f0/app/models/post.rb#L10

Your Design
<img width="1436" alt="Screenshot 2023-11-03 at 16 18 51" src="https://github.com/Wineshuga/pen-it/assets/58692590/9e11b4d3-0a63-4ae6-8ea2-e49edf4d8bac">

Required Design
<img width="538" alt="Screenshot 2023-11-03 at 16 19 20" src="https://github.com/Wineshuga/pen-it/assets/58692590/80c088d3-1702-4e73-8b2d-a4e6b51fbb9b">